### PR TITLE
updated workflows and send form to use interactivity

### PR DESCRIPTION
**Pull request by stevengill**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [x] New feature
- [ ] Bug fix
- [ ] Documentation

### Summary

Added interactivity to trigger/workflow/send form as it is required now. Also updated send_form to use new schema. 

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ After creating your app, you'll need to create a new trigger that will start the
 slack triggers create --trigger-def ./trigger.ts
```
Comment (mcodik) [inline README.md]: plz update to say "trigger create",  the singular form

Comment (stevengill) [reply README.md]: Fixed!

**Review by WilliamBergamin — APPROVED** [review]
lgtm 🚢

**Review by shapirone — APPROVED** [review]
LGTM

**Pull request merged**